op-e2e/system_fpp_test.go

```diff
@@ -96,19 +209,38 @@ func testVerifyL2OutputRoot(t *testing.T, detached bool) {
 	require.NoError(t, err, "get l1 head block")
 	l1Head := l1HeadBlock.Hash()
 
+	testFaultProofProgramScenario(t, ctx, sys, &FaultProofProgramTestScenario{
+		L1Head:             l1Head,
+		L2Head:             l2Head,
+		L2Claim:            common.Hash(l2Claim),
+		L2ClaimBlockNumber: l2ClaimBlockNumber,
```

Comment:
I think we should be passed `detached` through here too.

Reply:
whoops. thanks.